recipes/crisprmatchtest/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+  name: crisprmatch
+  version: "v1.0"
+
+source:
+  url: https://github.com/zhangtaolab/CRISPRMatch/archive/v1.0.tar.gz
+  sha256: 1aa546c7ca801467b29da2d50ec267ace7e96c5dc6a18aadfb05273d8e8a9bda
+
+build:
+  noarch: python
+
+requirements:
+  build:
+    - git
```

Comment:
why it git here needed? Can you inlude pip and remove setuptools if it is not needed?